app/resources/css-default.template

```diff
@@ -177,11 +177,16 @@ div.languages {
   text-decoration: none;
   color: black;
   text-align: {{end}};
+  float:right;
+  margin-right:5%;
```

Comment:
Okay. I am doing this.
One more thing->
I believe there unnecessary width:100% in .header class of sidebar.css file. For this, in my local machine there's unnecessary horizontal scroll bar at the bottom of the page. Without width:100% it works fine. could you please look at it.